hs_model_program/forms.py

```diff
@@ -76,11 +76,10 @@ def __init__(self, allow_edit=True, res_short_id=None, element_id=None, *args, *
         super(mp_form, self).__init__(*args, **kwargs)
         self.helper = mp_form_helper(allow_edit, res_short_id, element_id, element_name='MpMetadata', files=files)
 
+        # hide the field help text
         for field in self.fields:
```

Comment:
@pkdash I am looping over all of the fields and suppressing the help_text so that it is not displayed on the landing page.  I can simplify it to:
```
for field in self.fields:
    self.fields[field].help_text = None
```

I can remove the loop by also removing the help_text attributes from each field in models.py. Which do you think is better?